proto/caraml/upi/v1/prediction_log.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package caraml.upi.v1;
+
+import "caraml/upi/v1/variable.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/struct.proto";
+
+// PredictionLog stores information of prediction request handled by a specific model version.
+message PredictionLog {
+    // Unique identifier of prediction.
+    string prediction_id = 1;
+    // Target name / concept to be predicted by the prediction.
+    string target_name = 2;
+    // Project name that host the model performing prediction.
+    string project_name = 3;
+    // Model name perforing the prediction.
+    string model_name = 4;
+    // Model version performing the prediction.
+    string model_version = 5;
+
+    // Input of the prediction process 
+    ModelInput input = 10;
+    // Output of the prediction process
+    ModelOutput output = 11;
+
+    // Timestamp of the corresponding prediction request
+    google.protobuf.Timestamp request_timestamp = 20;
+
+    // Schema version of raw_features, features, entities, and prediction results fields are formatted
+    // I.e. for version 1, all of those fields will be formatted as a modified JSON SPLIT representation of a table
+    uint32 table_schema_version = 100;    
```

Comment:
will this updated when new field is introduced in this proto?

Reply:
No it won't it's just updated when the new encoding of table is introduced.